qcodes/instrument_drivers/yokogawa/GS200.py

```diff
@@ -1,5 +1,168 @@
-from qcodes.instrument.visa import VisaInstrument
-from qcodes.utils.validators import Numbers
+from typing import Optional
+
+from functools import partial
+
+from qcodes import VisaInstrument, InstrumentChannel
+from qcodes.utils.validators import Numbers, Bool, Enum, Ints
+
+
+def float_round(val):
+    """
+    Rounds a floating number represented as a string
+
+    Parameters
+    ----------
+    val: str
+
+    Returns
+    -------
+    int
+    """
+    return round(float(val))
+
+
+class GS200Exception(Exception):
+    pass
+
+
+class GS200_Monitor(InstrumentChannel):
+    """
+    Monitor part of the GS200. This is only enabled if it is
+    installed in the GS200 (it is an optional extra).
+
+    The units will be automatically updated as required.
+
+    To measure:
+    `GS200.measure.measure()`
+
+    Args:
+        parent (GS200)
+        name (str): instrument name
+        present (bool):
+    """
+    def __init__(self, parent: GS200, name: str, present: bool):
```

Comment:
`GS200` is not defined yet at this point. I think `'GS200'` should do the trick.